fibonacci.py

```diff
@@ -0,0 +1,11 @@
+def fibonacci(n):
+	l = [0,1]
+	for i in range(0, n+1):
+		if i > 1:
+			l.append(l[i-1] + l[i-2])
+		print(l[i])
+
+def test():
+	fibonacci(5)
+
+test()
```

Comment:
Thank you so much for your merge request. There are a couple of comments:
- performance: this isn't a very efficient way to calculate the fibonacci since we'll have to calculate each element multiple times. You might want to use memoization. See more here: https://stackoverflow.com/a/18172463
- style: please use spaces instead of tabs, since tabs get converted weirdly on GitHub.